tensorflow/core/kernels/list_kernels.h

```diff
@@ -664,9 +642,8 @@ class TensorListSplit : public OpKernel {
       // prevent this.
       Tensor aligned;
       OP_REQUIRES_OK(c, c->allocate_temp(tmp.dtype(), tmp.shape(), &aligned));
```

Comment:
>allocate_temp [](start = 27, length = 13)

It looks like TensorCopyUnaligned already calls allocate_temp. Do callers need to allocate their own tensors, or not?

Reply:
The callers need. Removed the extra line to call allocate_temp.

---
In reply to: [525578893](https://github.com/microsoft/tensorflow-directml/pull/95#discussion_r525578893) [](ancestors = 525578893)